# Diff: https://project-skyscraper.com/wp-json/wp/v2/pages?per_page=100
# File: api/wp-json/wp/v2/pages_per_page_100.json
# Timestamp: 2026-05-28 14:24:46 UTC
# Lines: 935 old -> 935 new
# Beautified: yes
--- old/api/wp-json/wp/v2/pages_per_page_100.json
+++ new/api/wp-json/wp/v2/pages_per_page_100.json
@@ -496,7 +496,7 @@
       "rendered": "Neural network status"
     },
     "content": {
-      "rendered": "\n<p class=\"wp-block-paragraph\"><span class=\"online-iterations \"><strong>4</strong> Inbound Attempts</span></p>\n",
+      "rendered": "\n<p class=\"wp-block-paragraph\"><span class=\"online-iterations \"><strong>3</strong> Inbound Attempts</span></p>\n",
       "protected": false
     },
     "excerpt": {
